<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UV-Vis &amp; TDDFT Plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="widget">
       <attribute name="title">
        <string>Exp. Data.</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tblExpData"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="btnExpDataPath">
            <property name="text">
             <string>Select Exp Data File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnReadExpData">
            <property name="text">
             <string>Read Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Data Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtExpDataDir"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>TDDFT</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tblExpData_2"/>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnTDPath">
            <property name="text">
             <string>Select Exp Data File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnReadTD">
            <property name="text">
             <string>Read Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Data Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtTDDir"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnConvert">
      <property name="text">
       <string>Plot Graph</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Exp. Data. Only</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TDDFT Only</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Both</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>새 항목</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
